fnet/cli/train_model.py

```diff
@@ -118,6 +118,9 @@ def main(args: Optional[argparse.Namespace] = None) -> None:
         train_options = json.load(fi)
     args.__dict__.update(train_options)
 
+    if not os.path.exists(args.path_save_dir):
```

Comment:
I don't think you need the if statement here. 
os.makedirs makes the directories if they don't exist. 
if they exist I don't think it has any effect.

Reply:
The function apparently now has an `exist_ok` parameter (https://docs.python.org/3/library/os.html#os.makedirs). This would make it unnecessary to check for directory existence.